app/package.json

```diff
@@ -34,7 +34,7 @@
     "build": "react-scripts build",
     "eject": "react-scripts eject",
     "gql-gen": "graphql-codegen --config codegen.js --require dotenv/config",
-    "lint": "eslint --ext ts,tsx src && tsc --noEmit && yarn run prettier",
```

Comment:
I personally prefer this style of type checking which doesn't actually output compiled JavaScript but understand if there's other stuff in `react-scripts build` worth incorporating in "lint".

Reply:
ah i see now - thanks, updated!